maker/src/main.rs

```diff
@@ -162,7 +162,7 @@ async fn main() -> Result<()> {
         projection::Actor::new(db.clone(), bitcoin_network, &price_feed);
     tasks.add(projection_context.run(proj_actor));
 
-    rocket::custom(figment)
+    let _ = rocket::custom(figment)
```

Comment:
🛠 I don't think we just want to silently drop this `Result` - shouldn't we at least log an `error` so we know what failed?

Reply:
It's still `?` at the end , so if an error is reported it will be thrown up.

Reply:
we're not dropping the `Result`, we're dropping `Rocket` struct after it's already in the orbit as we don't care about inspecting it after shutdown.

You can see more info here:
https://github.com/SergioBenitez/Rocket/issues/2168

Reply:
In order to reduce confusion, I have added a small commit that logs the Rocket state on TRACE level; it might become useful, but at least it's clear what we're doing here (and we're doing what the lib code indended us to do).